doc/source/whatsnew/v1.1.0.rst

```diff
@@ -215,6 +215,67 @@ MultiIndex
         # Common elements are now guaranteed to be ordered by the left side
         left.intersection(right, sort=False)
 
+- Bug in :meth:`MultiIndex.get_indexer` incorrectly handling use of pad and backfill options.  This is a regression introduced in pandas 0.23.0 (:issue:`29896`)
+
+*pandas [0.23.0, 1.1.0)*
```

Comment:
can you indicate that this is < 1.1.0 (the interval notation is too subtle here)

Reply:
done